TicketSwap/solar
# feat(icons): add sound on off icons

# Description

Added `SoundOn` and `SoundOff` icons to use in our video player for the event header.

## How to test

- Checkout this branch
- `$ yarn && yarn storybook`
- Go to icons, see how they look

## Screenshots

<img width="288" alt="image" src="https://user-images.githubusercontent.com/12168698/170055291-26f642d9-6641-4b2b-b9db-88c0793ee03f.png">

## To be notified

@TicketSwap/frontend
